src/sql/connection.py

```diff
@@ -336,3 +336,15 @@ def _close(cls, descriptor):
 
     def close(self):
         self.__class__._close(self)
+
+    def get_curr_connection_info(self):
+        """Returns the dialect, driver, and database server version info"""
+        if not self.current:
+            return None
+
+        engine = self.current.metadata.bind
+        return {
+            "dialect": engine.dialect.name,
```

Comment:
Will these three attributes always exist? to be safe, I think lets do `gettattr(engine.dialect, "name", None)` so it returns None if it doesn't exist. we don't want jupysql to fail because of this

Reply:
Good catch! Better to have some failback here

Fix in [9f14f44](https://github.com/ploomber/jupysql/pull/126/commits/9f14f441d253fe19779c703a8cbe2730f83b341c)